src/Service/OS2Forms/CaseSubmissionNormalizers/CaseSubmissionNormalizer.php

```diff
@@ -2,176 +2,140 @@
 
 namespace App\Service\OS2Forms\CaseSubmissionNormalizers;
 
+use App\Entity\Board;
+use App\Entity\ComplaintCategory;
 use App\Exception\WebformSubmissionException;
-use App\Repository\BoardRepository;
-use App\Repository\ComplaintCategoryRepository;
-use App\Repository\UserRepository;
-use App\Service\DocumentUploader;
 use Doctrine\ORM\EntityManagerInterface;
-use Symfony\Contracts\HttpClient\HttpClientInterface;
 
-class CaseSubmissionNormalizer implements SubmissionNormalizerInterface
+class CaseSubmissionNormalizer extends AbstractNormalizer implements SubmissionNormalizerInterface
 {
-    public function __construct(private BoardRepository $boardRepository, private DocumentUploader $documentUploader, private ComplaintCategoryRepository $complaintCategoryRepository, private string $selvbetjeningUserApiToken, private EntityManagerInterface $entityManager, private HttpClientInterface $httpClient, private UserRepository $userRepository)
-    {
-    }
-
     /**
      * Normalizes the properties shared by all case types.
      */
     public function normalizeSubmissionData(string $sender, array $submissionData): array
```

Comment:
Can and should be moved to parent class.

Reply:
Makes sense. Fixed.